src/plugins/discover/public/application/main/hooks/use_discover_state.ts

```diff
@@ -56,19 +56,11 @@ export function useDiscoverState({
   const stateContainer = useMemo(
     () =>
       getState({
-        getStateDefaults: () =>
-          getStateDefaults({
-            config: uiSettings,
-            data,
-            savedSearch,
-            storage,
-          }),
-        storeInSessionStorage: uiSettings.get('state:storeInSessionStorage'),
         history,
-        toasts: services.core.notifications.toasts,
-        uiSettings,
+        savedSearch,
```

Comment:
dear reviewer, this is the main change, it should be more readable now, the `getStateDefaults` that gets` getStateDefaults` is gone